chromium_src/chrome/browser/component_updater/widevine_cdm_component_installer.cc

```diff
@@ -192,9 +231,18 @@ CrxInstaller::Result WidevineCdmComponentInstallerPolicy::OnCustomInstall(
   return installer->WaitForCompletion();
 }
 
+bool WidevineCdmComponentInstallerPolicy::IsHttp404(
+    CrxInstaller::Result result) {
+  return result.error ==
+             static_cast<int>(update_client::InstallError::CUSTOM_ERROR_BASE) +
+                 static_cast<int>(net::ERR_HTTP_RESPONSE_CODE_FAILURE) &&
+         result.extended_error == 404;
+}
+
 WidevineArm64DllInstaller::WidevineArm64DllInstaller(
+    const GURL zip_url,
```

Comment:
nit: should either be `const GURL& zip_url` or `GURL zip_url` with `zip_url_(std::move(zip_url))`.

Reply:
Resolved in 6766ab6782693f4fe24b2ba717253993223daafb.